src/database/migrations/20190914142540-create-comment.js

```diff
@@ -0,0 +1,38 @@
+export default {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('Comments', {
+    id: {
+      allowNull: false,
+      autoIncrement: true,
+      primaryKey: true,
+      type: Sequelize.INTEGER
+    },
+    tripId: {
+      type: Sequelize.INTEGER,
+      allowNull: false,
+    },
+    userId: {
+      type: Sequelize.INTEGER,
+      allowNull: false,
+    },
+    comment: {
+      type: Sequelize.TEXT,
+      allowNull: false,
+    },
+    isVisible: {
```

Comment:
Please, why do we have this `isVisible` field?

Reply:
An admin might want to hide a comment